sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -251,12 +257,12 @@ const formatFormData = (
         unit.numBedrooms = null
     }
 
-    unit.floor = stringToNumber(unit.floor)
-    unit.maxOccupancy = stringToNumber(unit.maxOccupancy)
-    unit.minOccupancy = stringToNumber(unit.minOccupancy)
-    unit.numBathrooms = stringToNumber(unit.numBathrooms)
+    unit.floor = stringToNumberOrOne(unit.floor)
+    unit.maxOccupancy = stringToNumberOrOne(unit.maxOccupancy)
+    unit.minOccupancy = stringToNumberOrOne(unit.minOccupancy)
+    unit.numBathrooms = stringToNumberOrOne(unit.numBathrooms)
 
-    if (unit.sqFeet.length === 0) {
+    if (!unit.sqFeet) {
```

Comment:
Should you also keep the existing logic in place as well?
`if (!unit.sqFeet || unit.sqFeet.length === 0)`

Reply:
Actually, `!unit.sqFeet` should cover this, because the empty string is falsey.